packages/common/src/HardhatConfig.js

```diff
@@ -44,7 +44,15 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
     },
     ovm: { solcVersion: "0.8.4-broken_alpha" },
     networks: {
-      hardhat: { gas: 11500000, blockGasLimit: 11500000, allowUnlimitedContractSize: false, timeout: 1800000 },
+      hardhat: {
+        hardfork: "london",
```

Comment:
So just so I understand, we're going to begin testing against london once this PR goes in?

Reply:
correct, do you want to wait until mainnet switches over to merge this?